.gitignore

```diff
@@ -12,3 +12,7 @@ settings.py
 
 # Sphinx docs
 docs/_build/
+
+# IntelliJ
+.idea
```

Comment:
You probably want to also add these to your [global `.gitignore` file](https://davidwalsh.name/global-gitignore), so this doesn't have to be done for every project you contribute to.